azure-functions-language-worker-protobuf/.gitignore

```diff
@@ -0,0 +1,288 @@
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+##
+## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
+
+# User-specific files
```

Comment:
Hmm, this file was added automatically as a part of bringing in the protobuf repo (https://github.com/Azure/azure-functions-language-worker-protobuf)

This .gitignore is irrelevant since everything in the azure-functions-language-worker-protobuf directory here is .gitignored. Do you think we should take this out from the protobuf repo? Or is it ok to leave in as a benign side product?

Reply:
Yes. we should fix it in  protobuf repo.